kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/config/ConfigParser.java

```diff
@@ -84,7 +85,7 @@ public String toYaml(Configuration configuration) {
     }
 
     public static ObjectMapper createObjectMapper() {
-        return (ObjectMapper) new ObjectMapper(new YAMLFactory())
+        return (ObjectMapper) new ObjectMapper(new YAMLFactory().enable(YAMLGenerator.Feature.LITERAL_BLOCK_STYLE))
```

Comment:
what's the motivation for this part of the change?  No objection, just curiosity.

Reply:
Wasn't supposed to be included, it was an attempt to make the `DerivedResourcesTest` more deterministic. As the trailing spaces in the expected or `out-*` files was significant and caused jackson/snakeYaml to emit `|-` rather than just `|` so tests were failing for readily apparent reason. This was attempting to get the config parser to to emit `|-` consistently, it didn't so should have been reverted.

Reply:
I have reverted this.